scripts/generate-docs.sh

```diff
@@ -21,16 +21,27 @@
 # Exit on error.
 set -e
 
+PKG_NAME=$1
```

Comment:
Please document what the new arguments mean now (and how you're supposed to call this script)

Reply:
also i thiiiiiink you could get the package name from the url prefix btw (it's the one inside `magenta-js/tree/master/($1)/src/`, but your call